front-end/src/components/Card/Card.js

```diff
@@ -1,23 +1,90 @@
-import React from 'react';
+import React, { useState, useEffect } from 'react';
+import moment from 'moment';
+import { WatchLater, ThumbUpAlt, ThumbUpAltOutlined } from '@material-ui/icons';
+import { useDispatch, useSelector } from 'react-redux';
+import { getData } from '../../redux/nodes/authentication/selectors';
+import { likePost, unlikePost } from '../../redux/nodes/entities/posts/actions';
+import { getComments } from '../../redux/nodes/entities/comments/selectors';
+import { getUsers } from '../../redux/nodes/entities/users/selectors';
 import { StyledCard, StyledCardContent } from './styled';
 import { propTypes, defaultProps } from './props';
+import { Comment } from './Comment';
+import { GridContainer, GridItem } from '../Grid';
 import { Text } from '../Text';
+import { Button } from '../Button';
+import { CreateCommentForm } from '../Forms';
+
+export const Card = ({ post, ...rest }) => {
+  const dispatch = useDispatch();
+  const { userId } = useSelector(getData);
+  const { comments } = useSelector(getComments);
+  const { users } = useSelector(getUsers);
+  const [liked, setLiked] = useState(false);
+
+  useEffect(() => {
+    setLiked(post.likes.indexOf(userId) !== -1);
+  }, [post.likes]);
 
-export const Card = ({ children, ...rest }) => {
   return (
     <StyledCard {...rest}>
       <StyledCardContent>
-        <Text variant='h5' gutterBottom>
-          This is a title
-        </Text>
-        <Text variant='body2'>
-          Lorem ipsum dolor sit amet consectetur adipisicing elit. Totam saepe,
-          alias nesciunt possimus cupiditate a suscipit eligendi recusandae
-          harum pariatur numquam distinctio, quisquam sequi eveniet nostrum
-          libero accusamus mollitia. Culpa saepe id, eaque a sed distinctio
-          maiores nihil itaque accusantium eveniet eligendi assumenda
-          consequuntur sequi perferendis omnis, quo temporibus? Aut!
-        </Text>
+        <GridContainer>
+          <GridItem xs={12}>
+            <Text variant='h5' style={{ fontWeight: 'bold' }} gutterBottom>
+              {users[post.user].name}
+            </Text>
+          </GridItem>
+
+          <GridItem xs={12}>
+            <Text
```

Comment:
What text it is? Please rename it. Frome Example Title, Description

Reply:
The type of text is controlled using the variant prop. On each value to variant props different format of text is displayed.